exercises/largest-series-product/Example.cs

```diff
@@ -22,6 +36,7 @@ private static long ParseDigit(char c)
     {
         if (!char.IsDigit(c))
         {
+            Console.WriteLine("Invalid digit: " + c);

```

Comment:
Could you remove this line?

Reply:
Yes!

Reply:
I reverted all of my changes to `Example.cs` :)